package.json

```diff
@@ -24,5 +26,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
+    "sweetalert2": "^8.2.6"
```

Comment:
Wouldn't this also need to be a regular dependency?

Reply:
i don't think so. This is being used for the modals

Reply:
But if this application reached a production employment this package would not be installed. Would that cause breaking/unintended side effects?

Reply:
in that case you are right. So i will add this change too.